best-practices-storage-extension.md

```diff
@@ -0,0 +1,237 @@
+        "requester_pays": true
+      }
+    },
+    ...
+```
+
+Here, two types of schemas are defined: the public one through the CDSE initiative and the commercial one via CREODIAS, representing the commercial aspect of CDSE. This is an excellent example of how helpful the Storage Extension is and how it enables differentiation between free and billed connections. Additionally, both schemas have been enriched with an additional `description` attribute, which isn't included in the extension's standard set. However, thanks to the extension's flexibility, it can be added to better emphasize the differences between the two options.
+
+Example of an `aws-s3` connection with greater emphasis on URI-like platform definition:
+
+```json
+  ...
+  "properties": {
+    ...
+    "storage:schemas": {
+      "aws-std": {
+        "type": "aws-s3",
+        "platform": "https://{bucket}.s3.{region}.amazonaws.com",
+        "bucket": "naip-visualization",
+        "region": "us-west-2",
+        "requester_pays": true
+      }
+    },
+    ...
+  }
+  ...
+```
+
+This example demonstrates how the `platform` attribute can be defined as a URI template with placeholders (`{bucket}` and `{region}`), which are then resolved using the values provided in the `bucket` (another additional field) and `region` fields. Both examples are valid.
+
+`storage:schemas` can be defined at three levels:
+
+- Catalog
+- Collection
+- Item's Properties
+
+### `storage:refs`
+
+This attribute is implemented at the asset level, for both collections and items—meaning the actual physical files or data that the catalog references. `storage:refs` is a list of strings, where the possible elements are the keys previously defined in `storage:schemas`.
+
+Below is an example of how to communicate to users that an asset is available on multiple defined schemas:
+
+```json
+    "some_asset": { // asset level implementation
+      ...
+      "href": "s3://...",
+      "storage:refs": [
+        "cdse-s3",
+        "creodias-s3"
+      ],
+      ...
+    }
+```
+
+When a specific asset is available from only one provider:
+
+```json
+    "some_other_asset": { // asset level implementation
+      ...
+      "href": "s3://...",
+      "storage:refs": [
+        "aws-s3"
+      ],
+      ...
+    }
+```
+
+And an important case: **when some assets are served through an API and no S3 access has been provided**:
+
+```json
+    "yet_another_asset": { // asset level implementation
+      ...
+      "href": "https://...",
+      "storage:refs": [],
+      ...
+    }
+```
+
+However, its primary purpose is to describe s3 access that truly matters. For example, it's recommended to consider whether S3 access needs to be highlighted for public bucket URLs pointing to assets like manuals or other PDF files. There's a low likelihood that users will want to access public pdf documentation via the S3 standard.
+
+## Value Added by Combination with Other Extensions
```

Comment:
I really like this section 💯! I am wondering if the whole doc should be renamed to make it explicit that it is meant to cover the authentication and alternate assets as well. Maybe something like `best-practices-data-access.md` or just `best-practices-data-storage.md`?